meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/apalis-imx6/lmp-base.cfg

```diff
@@ -23,8 +36,7 @@ CONFIG_SPL_OF_CONTROL=y
 # CONFIG_SPL_USB_HOST_SUPPORT is not set
 # CONFIG_SPL_USB_GADGET is not set
 # CONFIG_SPL_DM_MMC is not set
-CONFIG_SYS_MMCSD_RAW_MODE_U_BOOT_SECTOR=0x100
-CONFIG_SECONDARY_BOOT_SECTOR_OFFSET=0x800
+CONFIG_IMX_HAB=y
```

Comment:
We usually enable IMX_HAB only for DISTRO=lmp-mfgtool

Reply:
Thanks, fixed.